doc/classes/RenderingDevice.xml

```diff
@@ -774,6 +774,14 @@
 				Returns [code]true[/code] if the [param texture] is valid, [code]false[/code] otherwise.
 			</description>
 		</method>
+		<method name="texture_native_handle">
+			<return type="int" />
+			<param index="0" name="texture" type="RID" />
+			<description>
+				Returns the internal graphics handle for this texture object. For use when communicating with 3rd party APIs mostly with third-party plugins.
```

Comment:
I meant using "third-party" instead of "3rd party", not instead of GDExtension ;)

Reply:
rofl :)